python/cuspatial/cuspatial/tests/test_from_geopandas.py

```diff
@@ -36,38 +36,42 @@ def test_dataframe_column_access(gs):
 
 def test_from_geoseries_complex(gs):
     cugs = cuspatial.from_geopandas(gs)
-    assert cugs.points.xy.sum() == 18
-    assert cugs.lines.xy.sum() == 540
-    assert cugs.multipoints.xy.sum() == 36
-    assert cugs.polygons.xy.sum() == 7436
-    assert cugs.polygons.polys.sum() == 38
-    assert cugs.polygons.rings.sum() == 654
+    assert cugs.points.xy.sum().as_py() == 18
+    assert cugs.lines.xy.sum().as_py() == 540
+    assert cugs.multipoints.xy.sum().as_py() == 36
+    assert cugs.polygons.xy.sum().as_py() == 7436
+    assert cugs.polygons.polys.sum().as_py() == 38
+    assert cugs.polygons.rings.sum().as_py() == 327
 
 
 def test_from_geopandas_point():
     gs = gpd.GeoSeries(Point(1.0, 2.0))
     cugs = cuspatial.from_geopandas(gs)
-    cudf.testing.assert_series_equal(cugs.points.xy, cudf.Series([1.0, 2.0]))
+    cudf.testing._utils.assert_eq(
+        cugs.points.xy, cudf.Series([1.0, 2.0], dtype="float64").to_arrow()
+    )
```

Comment:
Since methods defined in `_utils` is internal, using them is discouraged. Can we make it work by
```
cudf.testing.assert_series_equal(cugs.points.xy.to_pandas(), cudf.Series([1.0, 2.0], dtype="float64").to_pandas())
```

Or simply resort to `pd.testing.assert_series_equal`

Reply:
Done. I made the pyarrow array a Series.